tox.ini

```diff
@@ -146,6 +148,8 @@ setenv = GNOCCHI_STORAGE_DEPS=file
          GNOCCHI_TEST_DEBUG=1
 deps = {[testenv:docs]deps}
        sphinxcontrib-versioning
+# for < 4.3 doc
+       pbr
```

Comment:
do we need to upper cap pbr?

Reply:
I don't think so

Reply:
kk, just asking since it says new pbr will break us.